plugins/pulp_docker/plugins/registry.py

```diff
@@ -25,6 +25,7 @@ class V1Repository(object):
     DOCKER_ENDPOINT_HEADER = 'x-docker-endpoints'
     IMAGES_PATH = '/v1/repositories/%s/images'
     TAGS_PATH = '/v1/repositories/%s/tags'
+    API_VERSION_CHECK_PATH = '/v1/_ping'
```

Comment:
Austin wrote:
> API_V1_CHECK_PATH is better, this endpoint returns no information about v2.

Note that this is a class attribute of the V1Registry object, so v1 is
already scoped.

-- 
Randy Barlow
irc:   bowlofeggs